### DISCUSSION TITLE
Bun v1.1.12

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#11518] [Announcements]
This release fixes a regression briefly introduced in `bun upgrade` on Windows that occurred in Bun v1.1.11. There are no release notes, please take a look at the Bun v1.1.11 release notes instead.

To install Bun v1.1.12:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.12:
```bash
bun upgrade
```

### **[Read Bun v1.1.11's release notes on Bun's blog](https://bun.sh/blog/bun-v1.1.11)**

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.12'>Bun v1.1.12</a>.</em>

**Comment by mewisme** [comment]
Hey, I can't use the command *`bun upgrade`* in Windows. I always have to use the PowerShell command to reinstall it.

![image](https://github.com/oven-sh/bun/assets/63306485/a2425860-c9b1-4267-912f-88a15cc91de9)
